sdk/mobile-center-crashes/src/main/java/com/microsoft/azure/mobile/crashes/Crashes.java

```diff
@@ -694,6 +745,37 @@ void saveErrorLog(ManagedErrorLog errorLog, File errorStorageDirectory, String f
     }
 
     /**
+     * Enum for the processing status of crash in the last session.
+     */
+    private enum LastCrashProcessingStatus {
+        PREPARE_PROCESSING,
+        PROCESSING,
+        PROCESSED,
+        PROCESSING_FAILED,
+        NOT_FOUND
+    }
+
+    public interface LastCrashErrorReportListener {
```

Comment:
This is related to another comment on simplifying cases but we should not have this interface. We will have more data to return asynchronously like that in the near future such as installId for example.

We should add a generics callback in the core module with only 1 method onResult(T data) and data can be null (if not found), and should never fail on an exception.

Azme has that generic callback if you want to look at it.